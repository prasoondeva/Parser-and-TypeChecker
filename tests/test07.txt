{
	a : INT;
	b : INT;
	a = 1;
	b = 1;
		{ a : INT;
		a = 1;
		WHILE ( > a b )
		{
			a = - a b;
		}
	}
}