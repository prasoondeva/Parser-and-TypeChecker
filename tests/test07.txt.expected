a 4 2
b 5 3
a 7 6
a 8 6
b 8 3
a 10 6
a 10 6
b 10 3